tests/functional/test_firefox_dnt.py

```diff
@@ -0,0 +1,25 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+from selenium.common.exceptions import TimeoutException
+
+from ..pages.firefox_dnt import FirefoxDNTPage
+
+@pytest.mark.nondestructive
```

Comment:
Another PEP8 tip: top level functions should be surrounded with two blank lines. See http://legacy.python.org/dev/peps/pep-0008/#blank-lines for details. I would recommend using [flake8](https://pypi.python.org/pypi/flake8) as a precommit hook to check your code for compliance. We could also add a check in Travis CI so we avoid landing code that does not adhere.